aries-site/src/examples/templates/dashboards/DashboardExample.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
```

Comment:
So this will be the file which is displayed in the Prism code preview, correct?

If so, does this meet our criteria of "I should be able to copy and paste into my own code"? This certainly presents the cleanest and I can see the imports, but I just want to confirm we are agreement in going this direction.

This approach also gives us the reusability of `ScreenContainer`, `AppHeader`, and `PageHeader` which we will want to use in other Template examples.

Reply:
Great idea. I had forgotten we had discussed that as a solution. Creating https://github.com/hpe-design/design-system/issues/419 to represent this work.